packages/tasks/src/tasks/chat-completion/spec/output_stream.json

```diff
@@ -0,0 +1,48 @@
+{
+	"$id": "/inference/schemas/chat-completion/output.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "Chat Completion Stream Output",
+	"title": "ChatCompletionStreamOutput",
+	"type": "object",
+	"properties": {
+		"choices": {
+			"type": "array",
+			"title": "ChatCompletionStreamOutputChoice",
+			"description": "A list of chat completion choices.",
+			"items": {
+				"type": "object",
+				"properties": {
+					"delta": {
+						"type": "object",
+						"title": "ChatCompletionStreamOutputDelta",
+						"description": "A chat completion delta generated by streamed model responses.",
+						"properties": {
+							"content": {
+								"type": "string",
+								"description": "The contents of the chunk message."
+							},
+							"role": {
+								"type": "string",
+								"description": "The role of the author of this message."
+							}
+						}
+					},
+					"finish_reason": {
+						"$ref": "#/$defs/FinishReason",
```

Comment:
Interesting that this works while the definition is not in the file

If that works / is not too complex, I think it would be clearer to explicitly refer to the file where it is defined, like we do for classification outputs for ex:
https://github.com/huggingface/huggingface.js/blob/80698813868a11045616dc86ab2fdd1b34c493c4/packages/tasks/src/tasks/image-classification/spec/output.json#L7-L10

Reply:
Actually it works because all schemas (input, output, output_stream) are combined before generating code. But good idea yes, better to be explicit :+1: